docs/build-with-powerloom/use-cases/existing-implementations/uniswap-dashboard/index.md

```diff
@@ -18,13 +18,13 @@ In essence, Pooler is to Uniswap and other DeFi product what Forex platforms are
 
 
 ### How it works
-Let's explore the details of the Pooler implementation. The image provided illustrates the operational flow of the UniswapV2 Dashboard hosted at [uniswapv2.powerloom.io](https://uniswapv2.powerloom.io).
+Let's explore the details of the Pooler implementation. The image provided illustrates the operational flow of the UniswapV2 Dashboard hosted at [uniswapv2.powerloom.io](https://uniswapv2.powerloom.io), and the same workflow is used for the UniswapV3 Dashboard hosted at [uniswapv3.powerloom.io](https://uniswapv3.powerloom.io).
```

Comment:
Let's put this part in a callout section else the text and graphics after this might confuse people since they solely have references to Uniswap V2.

Or rewrite the paragraph so that V3 is included appropriately.

Reply:
The V3 reference has been moved to a callout here:

https://github.com/PowerLoom/docs/commit/c96c956769b9e380ee1cf202e2fe351119da5cda